ppx_mysql.opam

```diff
@@ -17,9 +17,10 @@ build: [
 ]
 depends: [
   "alcotest" {with-test & >= "0.8" & < "0.9"}
-  "dune" {build}
-  "ocamlformat" {with-test & >= "0.9"}
+  "dune" {>= "1.4"}
```

Comment:
What is the reason for removing the `build` declaration for the `dune` package?

Reply:
opam-repository does not consider `dune` to be a `build` dependency anymore and has removed that annotation from all projects, because different dune versions might compile and install different files. Also the dune-generated `opam` files add the correct dune dependency automatically, but they don't have a `build` tag on them either, so there is consensus that `dune` is not considered a build dependency. You can see that [ppx_mysql 1.1 does not have a `build` dependency on `dune` anymore](https://github.com/ocaml/opam-repository/blob/b9d307f467203c91836b32ab30a8a7fe8dab635e/packages/ppx_mysql/ppx_mysql.1.1/opam#L19).

This change is to be forward compatible and prevent the discussion "can you remove `build` from `dune`" when submitting a the next release.

Reply:
Ok, thanks for the clarification! :-)